cmd/common-methods_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * MinIO Client (C) 2019 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+func TestGetDecodedKey(t *testing.T) {
+	getDecodeCases := []struct {
+		input  string
+		output string
+		err    error
+		status bool
+	}{
+		//success scenerio the key contains non printable (tab) character as key
```

Comment:
s/scenerio/scenario across this test case

Reply:
This needs to be fixed in more than one place. Please fix them everywhere

Reply:
done.